bundles/org.openhab.binding.irobot/README.md

```diff
@@ -12,18 +12,21 @@ without any need for a dedicated MQTT server.
 
 ## Discovery
 
-Roombas on the same network will be discovered automatically, however in order to connect to them a password is needed. The
-password is a machine-generated string, which is unfortunately not exposed by the original iRobot smartphone application,
-but it can be downloaded from the robot itself. If no password is configured, the Thing enters "CONFIGURATION PENDING" state.
-Now you need to perform authorization by pressing and holding the HOME button on your robot until it plays series of tones
-(approximately 2 seconds). The Wi-Fi indicator on the robot will flash for 30 seconds, the binding should automatically
-receive the password and go ONLINE.
-
-After you've done this procedure you can write the password somewhere in case if you need to reconfigure your binding. It's
-not known, however, whether the password is eternal or can change during factory reset.
+Roombas on the same network will be discovered automatically, however in order to connect to them a password is needed.
+The password is a machine-generated string, which is unfortunately not exposed by the original iRobot smartphone application, but it can be downloaded from the robot itself.
+If no password is configured, the Thing enters "CONFIGURATION PENDING" state.
+Now you need to perform authorization by pressing and holding the HOME/DOCK button on your robot until it plays series of tones (approximately 2 seconds).
+The Wi-Fi indicator on the robot will flash for 30 seconds, the binding should automatically receive the password and go ONLINE.
+
+After you've done this procedure you can write the password somewhere in case if you need to reconfigure your binding.
+It's not known, however, whether the password is eternal or can change during factory reset.
 If you have issues getting the password make sure there are no other devices like your smartphone communicating with the robot.
 You can also try using [these python scripts](https://github.com/NickWaterton/Roomba980-Python) to get the password.
 
+*note* For file based configuration storing the password is essential.
```

Comment:
I'm not sure how this is best formatted, but probably capitalize first letter and add a colon before the capital "F"?
```suggestion
*Note:* For file based configuration storing the password is essential.
```

Reply:
looked at some other bindings and by far the most use **NOTE**

Reply:
👍 I do think a colon is needed though as well, since the next word starts a new sentence.

Reply:
Done.